contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -827,6 +621,7 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension {
             sentAddresses[2] = sentAddresses[1]; // receiver = borrower
         }
 
+        loanDataBytes = _checkPermit(collateralTokenAddress, loanDataBytes);
```

Comment:
this only works for borrow and not for margin trade as margin trade can be funded with either loan token or collateral token. This needs to be re-designed to check permit for whichever token is being used.

Reply:
agree that will require some thinking. TODO

Reply:
still todo